src/org/openj9/envInfo/MachineInfo.java

```diff
@@ -101,7 +116,78 @@ private String execCommands(String[] commands) {
 		} catch (IOException | InterruptedException e) {
 			e.printStackTrace();
 		}
-		return rt;
+		if (rt == null){
```

Comment:
You can have `rt = "Command could not be executed";`  in the catch clause, so you don't need the if check in the end.